input/ig.xml

```diff
@@ -259,10 +259,10 @@
 		</resource>
 		<resource>
 			<reference>
-				<reference value="Condition/cellulitis"/>
+				<reference value="Condition/condition-masked"/>
 			</reference>
-			<name value="Condition - Cellulitis of Right Knee"/>
-			<description value="Cellulitis of right knee&#xa;&#xa;Patient: Anne Bennelong"/>
+			<name value="Condition - Example with masked code"/>
```

Comment:
Title and describe this as supressed data.

Reply:
change not in PR yet.

Reply:
Please check again